added concepts (#135)
pytorch-ignite/pytorch-ignite.ai: src/concepts/_index.md

---
title: Concepts
weight: 3
---

# Concepts

> Understand the essence of the library in detail and why things are built a certain way.

Reviewer: ```suggestion
Understand the essence of the library in detail and why things are built in this way.
```
